sdk/core/azure-core-test/src/main/java/com/azure/core/test/InterceptorManager.java

```diff
@@ -368,4 +430,20 @@ private File createRecordFile(String testName) throws IOException {
     public void addTextReplacementRule(String regex, String replacement) {
         textReplacementRules.put(regex, replacement);
     }
+
+    /**
+     * Add text replacement rule (regex as key, the replacement text as value) into {@code recordSanitizers}
+     * @param testProxySanitizers the list of replacement regex and rules.
+     */
+    public void addSanitizers(List<TestProxySanitizer> testProxySanitizers) {
```

Comment:
It would be good to have this be called `setSanitizers` as this will overwrite any existing sanitizers, comment applies for `addMatchers` as well.

Reply:
Hmm...They "add" to the existing "default" sanitizers/matchers. Do we still want to call them `set`?

Reply:
The new impl here is additive.